FaH instead of Faucet

nformant1 opened the pull request:
Removed mining, only "mining" in "get some..." remained. Changed faucets to FoldingAtHome and added missing btn-outline-secondary.

Reason: All faucets are down.

patricklodder commented:
Hey! In general, I think that with all the faucets being defunct, this does offer an alternative for shibes to earn DOGE, and potentially "do good".

2 things that I noticed:

- I am not sure if FaH is "the easiest way", but maybe it's "another way"?
- Can we rename the i18n identifiers to make it reflect "FaH" and not faucets?

nformant1 commented:
Changed and uploaded, I also added a small OSX/python how to (link is pointing at the dogecoin rep, so its not working on my fork) - looking forward to read your feedback (:

patricklodder commented on assets/i18n/en.json:
```diff
@@ -93,11 +93,11 @@
+    "dgc-exchanges-modal-tipped": "You can get tipped Dogecoin by participating in <a class=\"btn btn-sm btn-outline-secondary\" target=\"_blank\" href=\"http://www.reddit.com/r/dogecoin\">our community</a>. If that's not your style, another way to have your first Dogecoin is to participate at Folding@Home.", 
+    "dgc-exchanges-modal-fah-faq": "<a class=\"btn btn-sm btn-outline-secondary\" target=\"_blank\" href=\"https://www.dogecoinfah.com\" target=\"_blank\">Folding@Home (FaH)</a> is a distributed computing project. You can join team dogefolders, bring in your computing power, help to find find cures for diseases and receive some Doges for your efforts.", 
+    "dgc-exchanges-modal-faucet-sites": "Here are a all resources to start folding:", 
+    "dgc-exchanges-modal-dogecoinfah": "DogecoinFaH.com", 
+    "dgc-exchanges-modal-dogecoinfah-desc": " - Team dogefolders.", 
```
I think those tags are no longer implemented in the html. Can we remove them? Also for other translations.

patricklodder commented on assets/i18n/doge.json:
```diff
@@ -93,8 +93,8 @@
     "dgc-exchanges-modal-many-ways": "There are numerous amounts of ways to get Dogecoin. You can buy them, trade for them, get tipped, \"mine\" them, and more. They are surprisingly endless!", 
-    "dgc-exchanges-modal-tipped": "You can get tipped Dogecoin by participating in <a href=\"http://www.reddit.com/r/dogecoin\">our community</a>. If that's not your style, the easiest way to have your first Dogecoin is to get them from a faucet.", 
-    "dgc-exchanges-modal-faucet-faq": "A Faucet is a website which gives you a small amount of Dogecoin for free to introduce you to the currency! All you need is your Dogecoin address.", 
+    "dgc-exchanges-modal-tipped": "You can get tipped Dogecoin by participating in <a class=\"btn btn-sm btn-outline-secondary\" target=\"_blank\" href=\"http://www.reddit.com/r/dogecoin\">our community</a>. If that's not your style, the easiest way to have your first Dogecoin is to participate at Folding@Home.", 
```
For consistency: missed change of "the easiest" into "another".

patricklodder commented on index.html:
```diff
@@ -215,13 +215,15 @@ <h2 data-i18n="dgc-getting-started" class="page-section-heading text-center text
                         <span data-i18n="dgc-exchanges-modal-tipped">You can get tipped Dogecoin by participating in <a class="btn btn-sm btn-outline-secondary js-scroll-trigger" href="#about">our community</a>
                             <br>
-                        If that's not your style, the easiest way to have your first Dogecoin is to get them from a faucet.</span>
+                        If that's not your style, the easiest way to have your first Dogecoin is to participate at Folding@Home.</span>
```
Does not reflect English translation.

patricklodder reviewed (changes requested):
Thanks for the guide. Would normally ask for a separate PR but this is fine because there's not so many changes. 

There are still some minor things that still need to be resolved, then we're good.

patricklodder reviewed (approved):
This looks good to me. Reviewed the source and the result in each language.
